src/main/java/uk/gov/cslearning/record/service/scheduler/LearningJob.java

```diff
@@ -64,6 +72,82 @@ public LearningJob(UserRecordService userRecordService, IdentityService identity
         this.notificationRepository = notificationRepository;
     }
 
+     void CheckAndNotifyLineManager(CivilServant civilServant, Identity identity,Course course, LocalDateTime completedDate) throws NotificationClientException {
+
+        if (civilServant.getLineManagerEmail() == null) {
+            LOGGER.error("User has no line manager!");
+        } else {
+            boolean sendMail = false;
+            Optional<Notification> optionalNotification = notificationRepository.findFirstByIdentityUidAndCourseIdAndNotificationTypeOrderBySentDesc(identity.getUid(),course.getId(),COMPLETED);
+
+            if (!optionalNotification.isPresent()) {
+                sendMail = true;
+            } else {
+                Notification notification = optionalNotification.get();
+                if (notification.getSent().isBefore(completedDate)) {
+                    sendMail = true;
+                }
+
+            }
+
+            if (sendMail) {
+                Optional<CivilServant> optionalLineManager = registryService.getCivilServantByUid(civilServant.getLineManagerUid());
+                if (optionalLineManager.isPresent()) {
+                    CivilServant lineManager = optionalLineManager.get();
+                    notifyService.notifyOnComplete(civilServant.getLineManagerEmail(), govNotifyCompletedLearningTemplateId, civilServant.getFullName(), lineManager.getFullName(), course.getTitle());
+                    Notification notification = new Notification(course.getId(), identity.getUid(), COMPLETED);
+                    notificationRepository.save(notification);
+                } else {
+                    LOGGER.error("User has line manager but line manager does not exist!");
+                }
+            }
+
+        }
+
+    }
+    public void sendNotificationForCompletedLearning() throws NotificationClientException {
+
+        Collection<Identity> identities = identityService.listAll();
+
+        for (Identity identity : identities) {
+
+            LOGGER.info("Got identity with uid {} ({})", identity.getUsername(), identities.size());
```

Comment:
`Got identity with uid identity-manager@domain.com (6)`

maybe reword this or don't show size?

Reply:
Fair enough. Made it clearer that that is a total.